api-spec/v1/openapi.yaml

```
@@ -25,7 +25,7 @@ paths:
       description: 'GET login endpoint with reCAPTCHA code'
       parameters:
         - in: header
-          name: x-rptid
+          name: x-rpt-id

```

```suggestion
          name: x-rpt-ids

```
🚲  policy side we are populating this fields with a list of rpt ids (from 1 to 5) so a plural name will better fit here.
what do you think=

---

I agree with you. Done here 2dad9384292bba380ff425ce07508ecc11148f58